unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/SceneInfo/SceneLimitInfoController.cs

```diff
@@ -0,0 +1,158 @@
+using DCL;
+using DCL.Controllers;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class SceneLimitInfoController : MonoBehaviour
+{
+    [Header("Design")]
+    public Color lowFillColor, mediumFillColor, highFillColor;
+
+    [Header("Scene references")]
+
+    public GameObject landHoverBtn;
+    public TextMeshProUGUI[] titleTxt;
+    public TextMeshProUGUI leftDescTxt,rightDescTxt;
+    public Image[] fillsImgs;
+
+    ParcelScene currentParcelScene;
+
+
+    public void SetParcelScene(ParcelScene parcelScene)
+    {
+        currentParcelScene = parcelScene;
+        UpdateInfo();
+    }
+
+    public bool IsActive()
+    {
+        return gameObject.activeSelf;
+    }
+
+    public void Enable()
+    {
+        gameObject.SetActive(true);
+        landHoverBtn.SetActive(false);
+        UpdateInfo();
+    }
+
+    public void Disable()
+    {
+        gameObject.SetActive(false);
+        landHoverBtn.SetActive(true);
+    }
+
+    public void UpdateInfo()
+    {
+        if (IsParcelSceneSquare(currentParcelScene))
+        {
+            int size = (int)Math.Sqrt(currentParcelScene.sceneData.parcels.Length);
+            int meters = size * 16;
+            titleTxt[0].text = size + "x" + size + " LAND <color=#959696>" + meters + "x" + meters + "m";
+            titleTxt[1].text = size + "x" + size + " LAND";
+        }
+        else
+        {
+            titleTxt[0].text = "CUSTOM LAND";
+            titleTxt[1].text = "CUSTOM LAND";
+        }
+
+
+        SceneMetricsController.Model limits = currentParcelScene.metricsController.GetLimits();
+        SceneMetricsController.Model usage = currentParcelScene.metricsController.GetModel();
+
+        string leftDesc = AppendUsageAndLimit("ENTITIES", usage.entities, limits.entities);
+        leftDesc += "\n" + AppendUsageAndLimit("BODIES", usage.bodies, limits.bodies);
+        leftDesc += "\n" + AppendUsageAndLimit("TRIS", usage.triangles, limits.triangles);
+        string rightDesc = AppendUsageAndLimit("TEXTURES", usage.textures, limits.textures);
+        rightDesc += "\n" + AppendUsageAndLimit("MATERIALS", usage.materials, limits.materials);
+        rightDesc += "\n" + AppendUsageAndLimit("GEOMETRIES", usage.meshes, limits.meshes);
+      
+        
+       
+        leftDescTxt.text = leftDesc;
+        rightDescTxt.text = rightDesc;
+        SetFillInfo();
+    }
+
+    void SetFillInfo()
+    {
+        float percentAmount = GetHigherLimitPercentInfo();
+        Color colorToUse = lowFillColor;
+        if (percentAmount > 66)
+            colorToUse = mediumFillColor; 
+        if (percentAmount > 85)
+            colorToUse = highFillColor; 
```

Comment:
Yes! No problem, I usually expose these values to the Unity editor so it can be changed by designers. Will you like this approach or do you prefer to set them as constant in the Configuration file?

Reply:
I like the approach of expose them as editor values 👍

Reply:
@Suduck finally, didn't you extract these values (66, 85...) as inspector variables? I notice that the values continue being hardcoded in the code.